frontend/src/metabase/visualizations/lib/settings/series.js

```diff
@@ -184,8 +184,7 @@ export function seriesSetting({ readDependencies = [], def } = {}) {
       inline: true,
       getHidden: (single, seriesSettings, { settings, series }) =>
         series.length <= 1 || // no need to show series-level control if there's only one series
-        !settings["graph.show_values"] || // don't show it unless this chart has a global setting
-        settings["stackable.stack_type"], // hide series controls if the chart is stacked
+        !settings["graph.show_values"], // don't show it unless this chart has a global setting
```

Comment:
It would be nice to keep this setting hidden when `Stack values to show` is set to `Total` as in this case we do not show individual series data labels

Reply:
Ah that's true